rails/spec/controllers/admin/project_links_controller_spec.rb

```diff
@@ -6,6 +6,7 @@
 
 describe Admin::ProjectLinksController do
   before(:each) do
+    generate_default_settings_and_jnlps_with_mocks
```

Comment:
Can you explain why this is necessary?

Reply:
It isn't strictly necessary, but without out there is a message printed during the test about "no active settings" or something like that. So I added this to get rid of that noise.